DangerRulesTest/ViewController.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 class ViewController: UIViewController {
     
     var testVar:String = "abc"
+    
+    var testVarTwo:Int = 123
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayur-gauswami/DangerIntegration/blob/e1e2d2fc76ba78d544509e9d920e5b42a9ef7b42/DangerRulesTest/ViewController.swift#L14"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals<br />
<code>colon</code> <code>ViewController.swift:14</code></td>
    </tr>
  </tbody>
</table>